kakao/src/main/kotlin/com/agoda/kakao/common/views/KBaseView.kt

```diff
@@ -76,6 +79,17 @@ open class KBaseView<out T> : BaseActions, BaseAssertions {
         function(this as T)
     }
 
+    fun configure(configuratorBuilderAction: ConfiguratorBuilder.() -> Unit) {
+        val configuratorBuilder = ConfiguratorBuilder.createWithHistory(KakaoConfigurator.configurator)
+        configuratorBuilderAction.invoke(configuratorBuilder)
+        view = configuratorBuilder.getConfigurator().viewInteractionDelegateFactory
+            .invoke(view.viewInteraction)
+    }
+
+    fun resetCustomConfigurator() {
```

Comment:
I think `reset` is enough with proper documentation.

Reply:
done